### PR TITLE
install prettier

### PR DESCRIPTION
**Pull request by jawiemkto**
install husky
.prettierrc.json: add config for prettier
husky generate pre-commit: run prettier --check
add postinstall stage, add prettier script

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname $0)/_/husky.sh"
```
Comment (oneacik) [inline .husky/pre-commit:2]: what is purpose of it?

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "bracketSpacing": true,
+  "trailingComma": "none",
```
Comment (oneacik) [inline .prettierrc.json:5]: what does trailing comma?

Comment (oneacik) [reply .prettierrc.json:5]: https://codeburst.io/learn-javascript-es-2017-trailing-commas-4534891f3f90

**Review by oneacik — APPROVED** [review]

File: .prettierrc.json (hunk 2 above)
Comment (jawiemkto) [reply .prettierrc.json:5]: Adding coma after last element of array or object. I don;t like it :p

File: .husky/pre-commit (hunk 1 above)
Comment (jawiemkto) [reply .husky/pre-commit:2]: Generated by husky, script executing husky.sh

Comment (jawiemkto) [reply .husky/pre-commit:2]: actually whole file is a pre-commit file, i mean it's also named pre-commit maybe it should be named prettier-stage?

File: package.json [hunk 3]
```diff
@@ -5,10 +5,12 @@
   "main": "index.js",
   "type": "commonjs",
   "scripts": {
+    "postinstall": "husky install",
```
Comment (jawiemkto) [inline package.json:8]: this script forces the installation of husky, to avoid situation that pre-commit won't work

File: .prettierrc.json (hunk 2 above)
Comment (oneacik) [reply .prettierrc.json:5]: me too

**Pull request merged**